SQL/0000-00-00-schema.sql

```diff
@@ -24,9 +24,11 @@ INSERT INTO subproject (title, useEDC, WindowDifference) VALUES
   ('Experimental', false, 'optimal');
 
 CREATE TABLE `project_rel` (
+  `ProjectSubprojectRelID` int(10) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
@ridz1208 Shouldn't it be ProjectRelID?
unless you also want to rename the table to 'project_subproject_rel'.
don't forget to adjust PK def if needed and subsequent code.

Reply:
it's already done in another PR. I just didnt wanna do the changes twice or mix the code

Reply:
Ok. reason accepted